### PR TITLE
Update all instances of GoPhish to Gophish

### LINKED ISSUE #35: Change all instances of GoPhish to Gophish
# 💡 Summary #

Anywhere in the gophish-tools repository where there is an instance or reference to GoPhish, change it to Gophish
## Motivation and context ##

Why does this work belong in this project?

This would keep the naming convention in line with other CISAGov projects using Gophish.

## Acceptance criteria ##

How do we know when this work is done?

- [x] All instances and references to GoPhish are changed to Gophish
- [x]  Gophish-tools still works as expected

### PR DESCRIPTION
**Pull request by ameliav**
## 🗣 Description ##

All files that reference GoPhish will have been changed to Gophish. Resolves https://github.com/cisagov/gophish-tools/issues/35

## 💭 Motivation and context ##

This would keep the naming convention in line with other CISAGov projects using Gophish.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -130,8 +130,8 @@ with the command `pca-wizard-templates  --targets`.
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-------:|:--------:|
 | name | Page name in the format of `{assessment identifier}-{integer}-{descriptor}` (e.g. "RV0000-1-AutoForward"). | string | | yes |
-| capture_credentials | Capture all non-password input with GoPhish. Supports forwarding after submit action. | boolean | | yes |
-| capture_passwords | Capture password input by the user, but note that captured input is **stored in plain text as of GoPhish version 0.9.0.** | boolean | `False` | no |
+| capture_credentials | Capture all non-password input with Gophish. Supports forwarding after submit action. | boolean | | yes |
+| capture_passwords | Capture password input by the user, but note that captured input is **stored in plain text as of Gophish version 0.9.0.** | boolean | `False` | no |
```
Comment (dav3r) [inline README.md:134]: This is outside the scope of this PR, but someone should confirm if captured passwords are still stored in plain text.  If so, this comment should be updated from `0.9.0.` to whatever the current Gophish version is.

**Review by dav3r — APPROVED** [review]
Note that if this PR description included `Resolves #35` or `Closes #35` instead of `Takes care of Issue #35`, then #35 would automatically be closed when this PR is merged.  For details, see [here](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue).

File: README.md (hunk 1 above)
Comment (dav3r) [reply README.md:134]: @ameliav - Can you either take care of this or create an issue about this?

Comment (ameliav) [reply README.md:134]: Sure, I will create an issue linking to your comment.

Comment (dav3r) [reply README.md:134]: Thanks! 👍

**Comment by ameliav** [comment]
This PR should be good to merge, thanks!

**Pull request merged**